network/data_link/wifi.c

```diff
@@ -59,7 +59,7 @@
 /******************************************************************************/
 
 /* Structure storing data used by a socket */
-struct socket_desc {
+struct wifi_sock {
```

Comment:
the name here is not consistent with the commit title. Should it be wifi_desc ?

Reply:
Changed in commit message